static/locales/en-US.yaml

```diff
@@ -871,6 +870,20 @@ Playlist:
   View: View
   Views: Views
   Last Updated On: Last Updated On
+  Sort By:
+    Sort By: Sort By
+    Custom: Custom
+    CustomDescending: Custom (descending)
```

Comment:
`Custom` already sound a bit weird to me
`Custom (descending)` makes it even more weird

I guess this means "user specified" order
`User-defined`? (Suggest a better if you can